src/modules/Dashboard/features/RespondentData/RespondentData.const.ts

```diff
@@ -12,6 +12,12 @@ export const defaultRespondentDataFormValues = {
   filterByIdentifier: false,
   identifier: [],
   versions: [],
+  selectedActivity: null,
```

Comment:
Also, please remove the following: `selectedActivity, identifiers, apiVersions, answers, responseOptions, and subscalesFrequency`. These entities have been moved to the `RespondentDataContextProvider` as well. Please refer to the attached screenshot.
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/5993c91e-66fb-4981-b829-b6750e2c2343)

Reply:
@IhorNesterenko99 thank you, fields have been removed. Remaining fields still referenced by [RespondentsDataFormValues](https://github.com/ChildMindInstitute/mindlogger-admin/blob/feature/M2-7189-multiinformant-metapod-dev-merge/src/modules/Dashboard/features/RespondentData/RespondentData.types.ts#L171)